app/src/main/java/com/gh4a/Gh4Application.java

```diff
@@ -136,9 +133,9 @@ public void onCreate() {
         prefs.registerOnSharedPreferenceChangeListener(this);
 
         boolean isDebuggable = (getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0;
-        sHasCrashlytics = !isDebuggable && !TextUtils.equals(Build.DEVICE, "sdk");
+        sHasCrashlytics = !isDebuggable && !TextUtils.equals(Build.DEVICE, "sdk") && !BuildConfig.FDROID;
```

Comment:
Why not move all this logic to the helper class as well? Just name it CrashReportingHelper instead of FabricApplication and move sNextUrlTrackingPosition and sHasCrashlytics there and then have this method there:

```
public static void init(Context context) {
    sHasCrashlytics = <...>
    if (sHasCrashlytics) {
        Fabric.with(context, new Crashlytics());
    }
}